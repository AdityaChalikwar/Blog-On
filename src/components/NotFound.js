import React from 'react'

const NotFoundPage = () => (
    <div>
        <h1>404 Error! Not Found</h1>
    </div>
)

export default NotFoundPage