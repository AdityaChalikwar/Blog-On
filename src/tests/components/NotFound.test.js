import {shallow} from 'enzyme'
import React from 'react'
import NotFound from '../../components/NotFound'

test('should render not found page', () => {
    const wrapper = shallow(<NotFound />)
    expect(wrapper).toMatchSnapshot()
})