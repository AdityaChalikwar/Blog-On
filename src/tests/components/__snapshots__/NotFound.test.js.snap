// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found page 1`] = `
<div>
  <h1>
    404 Error! Not Found
  </h1>
</div>
`;
